esp/applications/sql2ecl/esp.yaml

```diff
@@ -10,6 +10,7 @@ esp:
    enableSEHMapping: true
    httpConfigAccess: true
    logLevel: 1
+   maxRequestEntityLength: 80000000
```

Comment:
Why 80Mb? default used to be 60Mb as far as I can see? And you have now made the default 1Gb for some cases which is a huge jump...

Also, it might be better to support friendly values rather than having to count zeroes (there's a function in jlib to convert them).

It's not clear how (or if) these values can actually be overriden via helm - should there be something in values.yaml (and the schema) to provide some example of how to change the value